src/main/java/org/broadinstitute/hellbender/engine/SkeletonVariant.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+/**
+ * SkeletonVariant is a minimal implementation of the Variant interface.
+ */
+public class SkeletonVariant implements Variant {
+
+    private SimpleInterval interval;
+    private Boolean isSNP;
```

Comment:
Also prefer the primitive boolean to the class `Boolean` so we don't have to worry about nulls.